addons/mail/static/src/core/web/follower.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.Follower">
+        <DropdownItem class="'o-mail-Follower d-flex justify-content-between p-0'">
+            <a class="o-mail-Follower-details d-flex align-items-center flex-grow-1 px-3 o-min-width-0" t-att-class="{ 'o-inactive fst-italic opacity-25': !props.follower.is_active }" href="#" role="menuitem" t-on-click.prevent="(ev) => this.onClickDetails(ev)">
```

Comment:
I would use `.opacity-75.text-reset.text-decoration-line-through` instead of `.fst-italic.opacity-25`.

- italics is to be avoided, especially on Firefox some content is cut too frequently
- opacity 25 is too low, making it unreadable
- `text-reset` to not be grey rather than green, so it's more discrete than active followers.
- `text-decoration-line-through` to strikethrough, giving the right grasp the follower is archived.

Reply:
I didn't change the follower template, I moved it to a new component instead of a t-call.
Can we do the design changes in another PR?

Reply:
No worries, I can make the small changes myself quickly before r+ ;)

Reply:
Made the change, in addition to `gap-1` around followers otherwise the avatar `<img>` had no gap.